app/src/main/java/org/simple/clinic/drugs/EditMedicinesModel.kt

```diff
@@ -9,12 +9,13 @@ import java.util.UUID
 data class EditMedicinesModel(
     val patientUuid: UUID,
     val prescribedDrugs: List<PrescribedDrug>?,
-    val protocolDrugs: List<ProtocolDrugAndDosages>?
+    val protocolDrugs: List<ProtocolDrugAndDosages>?,
+    val openIntention: OpenIntention?
 ) : Parcelable {
 
   companion object {
     fun create(patientUuid: UUID): EditMedicinesModel {
-      return EditMedicinesModel(patientUuid, null, null)
+      return EditMedicinesModel(patientUuid, null, null, null)
```

Comment:
We should be passing the `OpenIntention` through the factory method here.

Reply:
I do that in the later commit

Reply:
Is there a point for doing that though? we are adding it in one commit and removing it in later commit without using it anywhere during that change. So we should've been making that change in a single commit.